code/datums/wires/_wires.dm

```diff
@@ -258,8 +258,13 @@
 		return TRUE
 
 	// Station blueprints do that too, but only if the wires are not randomized.
-	if(user.is_holding_item_of_type(/obj/item/blueprints) && !randomize)
-		return TRUE
+	if(!randomize)
+		if(user.is_holding_item_of_type(/obj/item/blueprints))
+			return TRUE
+		for(var/obj/item/photo/photo in user.held_items)
+			if(!photo.picture.has_blueprints)
```

Comment:
I think `picture` can be `null` for maploaded / maint photos?

Reply:
```suggestion
			if(!photo.picture || !photo.picture.has_blueprints)
```